ttvn/parse.py

```diff
@@ -0,0 +1,76 @@
+class Parser:
```

Comment:
General comment, I think there should also be a check for the text itself having it's own paragraph breaks that should also correlate to a paragraph here, even if it doesn't reach 6 sentences.

Reply:
You mean like a double linebreak?

Reply:
Yeah, probably the most likely scenario